modules/Rock.cmake

```diff
@@ -49,6 +49,29 @@ function(rock_add_compiler_flag_if_it_exists FLAG)
     endif()
 endfunction()
 
+macro(rock_add_target_properties _target _name)
+    get_target_property(_properties ${_target} ${_name})
+    if(NOT _properties)
+        set(_properties ${ARGN})
+    else()
+        list(APPEND _properties ${ARGN})
```

Comment:
Nope, you can't. If _properties doesn't exist and you try to append '-Wone' to it you will end up with _properties-NOTFOUND;-Wone.